Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -315,8 +342,10 @@ public final class PackageInfoMapper: PackageInfoMapping {
 
         return .init(
             platformToMinDeploymentTarget: minDeploymentTargets,
-            productToExternalDependencies: externalDependencies,
+            productsToTargetDependencies: productsToTargetDependencies,
+            testTargetsExternalDependencies: [:],
```

Comment:
why is this empty?

Reply:
oh this line need to be deleted. It is not needed anymore as tests target's dependencies are part of existing `productToExternalDependencies`. Will revert the parameter name change in next commit